edx_rest_framework_extensions/decorators.py

```diff
@@ -0,0 +1,40 @@
+""" Decorator functions. """
+from functools import wraps
+
+import waffle
+from edx_rest_framework_extensions.authentication import JwtAuthentication
+
+SWITCH_ENFORCE_JWT_SCOPES = 'oauth2.enforce_jwt_scopes'
```

Comment:
Taking lessons from edx-platform, it's useful to put all waffle declarations in a separate `config.py` file.